app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -674,6 +685,18 @@ protected void setupWithStatus(StatusViewData.Concrete status,
         }
     }
 
+    private void setupCounters(StatusViewData.Concrete status) {
+        if (countFavorites!=null)
+            countFavorites.setText(status.getFavouritesCount()>0?String.format(Locale.getDefault(),"%d",status.getFavouritesCount()):"");
+
+        if (countReposts!=null)
+            countReposts.setText(status.getReblogsCount()>0?String.format(Locale.getDefault(),"%d",status.getReblogsCount()):"");
+
+        if (countReplies!=null)
+            countReplies.setText(status.getRepliesCount()>0?String.format(Locale.getDefault(),"%d",status.getRepliesCount()):"");
```

Comment:
We use `numberFormat.format` elsewhere, i prefer that
I actually like showing the nuber more than  "1" "1+" .